cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -512,6 +512,21 @@ declare namespace Deno {
      *   },
      * });
      * ```
+     * 
+     * ```ts
+     * import { assertEquals } from "https://deno.land/std/testing/asserts.ts";
+     *
+     * Deno.test({
+     *   name: "inherit",
+     *   permissions: {
+     *     net: "inherit",
+     *   },
+     *   fn() {
+     *     const status = Deno.permissions.querySync({ name: "net" })
+     *     assertEquals(status.state, "granted");
+     *   },
+     * });
+     * ```
```

Comment:
Duplicating all the examples and adding sync versions for these properties seems to add a bit too much noise. Maybe we should only document the async versions and remove these additions?